package/gluon-mesh-babel/files/etc/sysctl.d/40-gluon-babel.conf

```diff
@@ -0,0 +1,2 @@
+# this is used for ecn in ipv6 and ipv4 networks
+net.ipv4.tcp_ecn=1
```

Comment:
This does not seem like it needs to be specific to Babel, maybe move this to gluon-core?